domain/src/DomainValidatorInterface.php

```diff
@@ -28,11 +28,13 @@ public function validate(DomainInterface $domain);
    *
    * @param \Drupal\domain\DomainInterface $domain
    *   A domain record.
+   * @param string $test_path
+   *   A path on the domain to test.
```

Comment:
Makes sense, but maybe can be tighter wording to stay on one line. Also needs the (optional) annotation.